src/resource/file.ts

```diff
@@ -139,6 +139,16 @@ type SaveFileOptions = WriteFileOptions & {
  * be rejected. You can initialise it first using [[createContainerAt]], or directly save the file
  * at the desired location using [[overwriteFile]].
  *
+ * This function is primarily useful if the current user does not access to change existing files in
+ * a Container, but is allowed to add new files; in other words, they have Append, but not Write
+ * access to a Container. This is useful in situations where someone wants to allow others to e.g.
```

Comment:
Personally I don't like the use of 'e.g.' in the middle of a sentence (it's Ok at the end of a sentence, or in brackets), so I'd just tweak this to:
```suggestion
 * access to a Container. This is useful in situations where someone wants to allow others to, for example,
```